### ISSUE TITLE
brightness adjust

### ISSUE THREAD

**Issue by eadmaster** [cnorlander/USB2VFD-Deck#1]
Do you know if is it possible to adjust/reduce brightness on these displays?

(I guess some specific serial cmd is required)

EDIT: according to the [manual here on page 32](https://www.matrixorbital.com/lk402-25), there is a "Set Brightness" command with hex `FE 99`

**Comment by eadmaster** [comment]
update: i've rewritten a bit of the [example logic using a thread to keep the screen updated](https://gist.github.com/eadmaster/3849be8c3a02e9f0b518bb0b6b811ddc), however the aforementioned set brightness command does not work, if you have some suggestion please let me know...

**Comment by cnorlander** [comment]
@eadmaster Yeah I suspect this is not possible due to the nature of the VFD display. Seems the LK402-25 has this command as it was intended for LCD but I suspect due to the higher voltage requirements for a VFD (30v+) they likely decided to drive the display off another chip rather than using the one integrated into the LK402-25 meaning its likely not adjustable.
